2013-12-01 23:32:26,303 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Entity] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Entity.establish(Entity.groovy:17)
	at groovyempire.Owner.establish(Owner.groovy:7)
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:24)
2013-12-01 23:32:26,309 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Entity] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Entity.establish(Entity.groovy:17)
	at groovyempire.Owner.establish(Owner.groovy:7)
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:24)
2013-12-01 23:32:26,311 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Entity] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Entity.establish(Entity.groovy:17)
	at groovyempire.Owner.establish(Owner.groovy:7)
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:24)
2013-12-01 23:32:26,311 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Entity] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Entity.establish(Entity.groovy:17)
	at groovyempire.Owner.establish(Owner.groovy:7)
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:24)
2013-12-01 23:32:26,326 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Entity] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Entity.establish(Entity.groovy:17)
	at groovyempire.Owner.establish(Owner.groovy:7)
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:24)
2013-12-01 23:32:26,326 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Entity] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Entity.establish(Entity.groovy:17)
	at groovyempire.Owner.establish(Owner.groovy:7)
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:24)
2013-12-01 23:33:42,987 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: name for class: groovyempire.banking.Bank
Possible solutions: all
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:30)
2013-12-01 23:33:43,000 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: name for class: groovyempire.banking.Bank
Possible solutions: all
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:30)
2013-12-01 23:33:43,007 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: name for class: groovyempire.banking.Bank
Possible solutions: all
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:30)
2013-12-01 23:33:43,010 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: name for class: groovyempire.banking.Bank
Possible solutions: all
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:30)
2013-12-01 23:33:43,045 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: name for class: groovyempire.banking.Bank
Possible solutions: all
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:30)
2013-12-01 23:33:43,048 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: name for class: groovyempire.banking.Bank
Possible solutions: all
	at groovyempire.BankSpec.Create bank account(BankSpec.groovy:30)
2013-12-01 23:41:06,302 [main] ERROR StackTrace  - Full Stack Trace:
Condition not satisfied:

entity.name == "Alidad1"
|      |    |
Alidad |    false
       |    1 difference (85% similarity)
       |    Alidad(-)
       |    Alidad(1)
       Alidad

	at groovyempire.EntitySpec.Create an owner(EntitySpec.groovy:26)
2013-12-01 23:41:06,310 [main] ERROR StackTrace  - Full Stack Trace:
Condition not satisfied:

entity.name == "Alidad1"
|      |    |
Alidad |    false
       |    1 difference (85% similarity)
       |    Alidad(-)
       |    Alidad(1)
       Alidad

	at groovyempire.EntitySpec.Create an owner(EntitySpec.groovy:26)
2013-12-01 23:41:06,310 [main] ERROR StackTrace  - Full Stack Trace:
junit.framework.AssertionFailedError: Condition not satisfied:

entity.name == "Alidad1"
|      |    |
Alidad |    false
       |    1 difference (85% similarity)
       |    Alidad(-)
       |    Alidad(1)
       Alidad

	at groovyempire.EntitySpec.Create an owner(EntitySpec.groovy:26)
2013-12-01 23:41:06,310 [main] ERROR StackTrace  - Full Stack Trace:
junit.framework.AssertionFailedError: Condition not satisfied:

entity.name == "Alidad1"
|      |    |
Alidad |    false
       |    1 difference (85% similarity)
       |    Alidad(-)
       |    Alidad(1)
       Alidad

	at groovyempire.EntitySpec.Create an owner(EntitySpec.groovy:26)
2013-12-01 23:41:06,311 [main] ERROR StackTrace  - Full Stack Trace:
junit.framework.AssertionFailedError: Condition not satisfied:

entity.name == "Alidad1"
|      |    |
Alidad |    false
       |    1 difference (85% similarity)
       |    Alidad(-)
       |    Alidad(1)
       Alidad

	at groovyempire.EntitySpec.Create an owner(EntitySpec.groovy:26)
2013-12-01 23:41:06,311 [main] ERROR StackTrace  - Full Stack Trace:
junit.framework.AssertionFailedError: Condition not satisfied:

entity.name == "Alidad1"
|      |    |
Alidad |    false
       |    1 difference (85% similarity)
       |    Alidad(-)
       |    Alidad(1)
       Alidad

	at groovyempire.EntitySpec.Create an owner(EntitySpec.groovy:26)
2013-12-01 23:52:25,096 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.establishRelationshipForCollection(DefaultGrailsDomainClass.java:362)
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.establishRelationships(DefaultGrailsDomainClass.java:262)
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.<init>(DefaultGrailsDomainClass.java:120)
	at org.codehaus.groovy.grails.commons.DomainClassArtefactHandler.newArtefactClass(DomainClassArtefactHandler.java:54)
	at org.codehaus.groovy.grails.commons.AnnotationDomainClassArtefactHandler.newArtefactClass(AnnotationDomainClassArtefactHandler.java:54)
	at org.codehaus.groovy.grails.commons.DefaultGrailsApplication.addArtefact(DefaultGrailsApplication.java:812)
	at org.codehaus.groovy.grails.commons.DefaultGrailsApplication.addArtefact(DefaultGrailsApplication.java:538)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.springsource.loaded.ri.OriginalClassInvoker.invoke(OriginalClassInvoker.java:47)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1283)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.junit.runners.Suite.runChild(Suite.java:127)
	at org.junit.runners.Suite.runChild(Suite.java:26)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.codehaus.groovy.grails.test.spock.GrailsSpecTestType.doRun(GrailsSpecTestType.groovy:149)
	at org.codehaus.groovy.grails.test.support.GrailsTestTypeSupport.run(GrailsTestTypeSupport.groovy:121)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runTests(GrailsProjectTestRunner.groovy:412)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.processTests(GrailsProjectTestRunner.groovy:392)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests_closure7(GrailsProjectTestRunner.groovy:310)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests(GrailsProjectTestRunner.groovy:296)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests(GrailsProjectTestRunner.groovy:211)
	at org.codehaus.groovy.grails.cli.fork.testing.ForkedGrailsTestRunner.runInstance(ForkedGrailsTestRunner.groovy:128)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProjectClassExecutor.run_closure1(ForkedGrailsProjectClassExecutor.groovy:59)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at groovy.lang.Closure.call(Closure.java:427)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProcess.startDaemon_closure3(ForkedGrailsProcess.groovy:129)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProcess.startDaemon(ForkedGrailsProcess.groovy:106)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProjectClassExecutor.run(ForkedGrailsProjectClassExecutor.groovy:57)
	at org.codehaus.groovy.grails.cli.fork.testing.ForkedGrailsTestRunner.main(ForkedGrailsTestRunner.groovy:75)
2013-12-01 23:52:25,097 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:52:25,098 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:52:25,099 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:52:25,100 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:52:25,100 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:07,120 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.establishRelationshipForCollection(DefaultGrailsDomainClass.java:362)
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.establishRelationships(DefaultGrailsDomainClass.java:262)
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.<init>(DefaultGrailsDomainClass.java:120)
	at org.codehaus.groovy.grails.commons.DomainClassArtefactHandler.newArtefactClass(DomainClassArtefactHandler.java:54)
	at org.codehaus.groovy.grails.commons.AnnotationDomainClassArtefactHandler.newArtefactClass(AnnotationDomainClassArtefactHandler.java:54)
	at org.codehaus.groovy.grails.commons.DefaultGrailsApplication.addArtefact(DefaultGrailsApplication.java:812)
	at org.codehaus.groovy.grails.commons.DefaultGrailsApplication.addArtefact(DefaultGrailsApplication.java:538)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.springsource.loaded.ri.OriginalClassInvoker.invoke(OriginalClassInvoker.java:47)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1283)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.junit.runners.Suite.runChild(Suite.java:127)
	at org.junit.runners.Suite.runChild(Suite.java:26)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.codehaus.groovy.grails.test.spock.GrailsSpecTestType.doRun(GrailsSpecTestType.groovy:149)
	at org.codehaus.groovy.grails.test.support.GrailsTestTypeSupport.run(GrailsTestTypeSupport.groovy:121)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runTests(GrailsProjectTestRunner.groovy:412)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.processTests(GrailsProjectTestRunner.groovy:392)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests_closure7(GrailsProjectTestRunner.groovy:310)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests(GrailsProjectTestRunner.groovy:296)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests(GrailsProjectTestRunner.groovy:211)
	at org.codehaus.groovy.grails.cli.fork.testing.ForkedGrailsTestRunner.runInstance(ForkedGrailsTestRunner.groovy:128)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProjectClassExecutor.run_closure1(ForkedGrailsProjectClassExecutor.groovy:59)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at groovy.lang.Closure.call(Closure.java:427)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProcess.startDaemon_closure3(ForkedGrailsProcess.groovy:129)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProcess.startDaemon(ForkedGrailsProcess.groovy:106)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProjectClassExecutor.run(ForkedGrailsProjectClassExecutor.groovy:57)
	at org.codehaus.groovy.grails.cli.fork.testing.ForkedGrailsTestRunner.main(ForkedGrailsTestRunner.groovy:75)
2013-12-01 23:56:07,121 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:07,122 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:07,122 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:07,122 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:07,123 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:48,693 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.establishRelationshipForCollection(DefaultGrailsDomainClass.java:362)
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.establishRelationships(DefaultGrailsDomainClass.java:262)
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.<init>(DefaultGrailsDomainClass.java:120)
	at org.codehaus.groovy.grails.commons.DomainClassArtefactHandler.newArtefactClass(DomainClassArtefactHandler.java:54)
	at org.codehaus.groovy.grails.commons.AnnotationDomainClassArtefactHandler.newArtefactClass(AnnotationDomainClassArtefactHandler.java:54)
	at org.codehaus.groovy.grails.commons.DefaultGrailsApplication.addArtefact(DefaultGrailsApplication.java:812)
	at org.codehaus.groovy.grails.commons.DefaultGrailsApplication.addArtefact(DefaultGrailsApplication.java:538)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.springsource.loaded.ri.OriginalClassInvoker.invoke(OriginalClassInvoker.java:47)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1283)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.junit.runners.Suite.runChild(Suite.java:127)
	at org.junit.runners.Suite.runChild(Suite.java:26)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.codehaus.groovy.grails.test.spock.GrailsSpecTestType.doRun(GrailsSpecTestType.groovy:149)
	at org.codehaus.groovy.grails.test.support.GrailsTestTypeSupport.run(GrailsTestTypeSupport.groovy:121)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runTests(GrailsProjectTestRunner.groovy:412)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.processTests(GrailsProjectTestRunner.groovy:392)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests_closure7(GrailsProjectTestRunner.groovy:310)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests(GrailsProjectTestRunner.groovy:296)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests(GrailsProjectTestRunner.groovy:211)
	at org.codehaus.groovy.grails.cli.fork.testing.ForkedGrailsTestRunner.runInstance(ForkedGrailsTestRunner.groovy:128)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProjectClassExecutor.run_closure1(ForkedGrailsProjectClassExecutor.groovy:59)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at groovy.lang.Closure.call(Closure.java:427)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProcess.startDaemon_closure3(ForkedGrailsProcess.groovy:129)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProcess.startDaemon(ForkedGrailsProcess.groovy:106)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProjectClassExecutor.run(ForkedGrailsProjectClassExecutor.groovy:57)
	at org.codehaus.groovy.grails.cli.fork.testing.ForkedGrailsTestRunner.main(ForkedGrailsTestRunner.groovy:75)
2013-12-01 23:56:48,694 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:48,694 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:48,695 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:48,695 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:48,696 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:54,874 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.establishRelationshipForCollection(DefaultGrailsDomainClass.java:362)
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.establishRelationships(DefaultGrailsDomainClass.java:262)
	at org.codehaus.groovy.grails.commons.DefaultGrailsDomainClass.<init>(DefaultGrailsDomainClass.java:120)
	at org.codehaus.groovy.grails.commons.DomainClassArtefactHandler.newArtefactClass(DomainClassArtefactHandler.java:54)
	at org.codehaus.groovy.grails.commons.AnnotationDomainClassArtefactHandler.newArtefactClass(AnnotationDomainClassArtefactHandler.java:54)
	at org.codehaus.groovy.grails.commons.DefaultGrailsApplication.addArtefact(DefaultGrailsApplication.java:812)
	at org.codehaus.groovy.grails.commons.DefaultGrailsApplication.addArtefact(DefaultGrailsApplication.java:538)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.springsource.loaded.ri.OriginalClassInvoker.invoke(OriginalClassInvoker.java:47)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1283)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.junit.runners.Suite.runChild(Suite.java:127)
	at org.junit.runners.Suite.runChild(Suite.java:26)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at org.codehaus.groovy.grails.test.spock.GrailsSpecTestType.doRun(GrailsSpecTestType.groovy:149)
	at org.codehaus.groovy.grails.test.support.GrailsTestTypeSupport.run(GrailsTestTypeSupport.groovy:121)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runTests(GrailsProjectTestRunner.groovy:412)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.processTests(GrailsProjectTestRunner.groovy:392)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests_closure7(GrailsProjectTestRunner.groovy:310)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests(GrailsProjectTestRunner.groovy:296)
	at org.codehaus.groovy.grails.test.runner.GrailsProjectTestRunner.runAllTests(GrailsProjectTestRunner.groovy:211)
	at org.codehaus.groovy.grails.cli.fork.testing.ForkedGrailsTestRunner.runInstance(ForkedGrailsTestRunner.groovy:128)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProjectClassExecutor.run_closure1(ForkedGrailsProjectClassExecutor.groovy:59)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at groovy.lang.Closure.call(Closure.java:427)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProcess.startDaemon_closure3(ForkedGrailsProcess.groovy:129)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProcess.startDaemon(ForkedGrailsProcess.groovy:106)
	at org.codehaus.groovy.grails.cli.fork.ForkedGrailsProjectClassExecutor.run(ForkedGrailsProjectClassExecutor.groovy:57)
	at org.codehaus.groovy.grails.cli.fork.testing.ForkedGrailsTestRunner.main(ForkedGrailsTestRunner.groovy:75)
2013-12-01 23:56:54,875 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:54,876 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:54,876 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:54,877 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-01 23:56:54,877 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.grails.exceptions.GrailsDomainException: Property [transactions] in class [class groovyempire.banking.Account] is a bidirectional one-to-many with two possible properties on the inverse side. Either name one of the properties on other side of the relationship [account] or use the 'mappedBy' static to define the property that the relationship is mapped with. Example: static mappedBy = [transactions:'myprop']
	at grails.test.mixin.domain.DomainClassUnitTestMixin.registerGrailsDomainClass(DomainClassUnitTestMixin.groovy:219)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomains(DomainClassUnitTestMixin.groovy:135)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:166)
	at grails.test.mixin.domain.DomainClassUnitTestMixin.mockDomain(DomainClassUnitTestMixin.groovy:165)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.runtime.extension.builtin.JUnitFixtureMethodsExtension$FixtureType$FixtureMethodInterceptor.intercept(JUnitFixtureMethodsExtension.java:145)
	at org.spockframework.runtime.extension.MethodInvocation.proceed(MethodInvocation.java:84)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
	at org.spockframework.util.ReflectionUtil.invokeMethod(ReflectionUtil.java:138)
2013-12-02 00:00:11,756 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Owner] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Owner$$EOOpqfZo.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:25)
2013-12-02 00:00:11,756 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Owner] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Owner$$EOOpqfZo.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:25)
2013-12-02 00:00:11,757 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Owner] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Owner$$EOOpqfZo.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:25)
2013-12-02 00:00:11,757 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Owner] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Owner$$EOOpqfZo.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:25)
2013-12-02 00:00:11,757 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Owner] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Owner$$EOOpqfZo.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:25)
2013-12-02 00:00:11,757 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.IllegalStateException: Method on class [groovyempire.Owner] was used outside of a Grails application. If running in the context of a test using the mocking API or bootstrap Grails correctly.
	at groovyempire.Owner$$EOOpqfZo.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:25)
2013-12-02 00:00:51,750 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpufEe.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:00:51,750 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpufEe.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:00:51,750 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpufEe.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:00:51,750 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpufEe.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:00:51,751 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpufEe.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:00:51,751 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpufEe.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:02:18,553 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:02:18,554 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:02:18,554 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:02:18,554 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:02:18,555 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:02:18,555 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:02,404 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrConstructorNewInstance(ReflectiveInterceptor.java:986)
	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:02,404 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:02,405 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:02,405 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:02,405 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:02,406 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxA1Q.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:47,987 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrConstructorNewInstance(ReflectiveInterceptor.java:986)
	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxXR2.establishAccount(Bank.groovy:31)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:47,987 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxXR2.establishAccount(Bank.groovy:31)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:47,987 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxXR2.establishAccount(Bank.groovy:31)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:47,988 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxXR2.establishAccount(Bank.groovy:31)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:47,988 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxXR2.establishAccount(Bank.groovy:31)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:03:47,989 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxLjE.establish(Account.groovy:20)
	at groovyempire.banking.Bank$$EOOpxXR2.establishAccount(Bank.groovy:31)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:05:27,871 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrConstructorNewInstance(ReflectiveInterceptor.java:986)
	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxxBo.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:05:27,872 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxxBo.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:05:27,872 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxxBo.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:05:27,872 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxxBo.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:05:27,873 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxxBo.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:05:27,873 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOpxxBo.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:08:10,449 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpydqy.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:08:10,453 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpydqy.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:08:10,459 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpydqy.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:08:10,463 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpydqy.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:08:10,474 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpydqy.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:08:10,477 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpydqy.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:09:17,361 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:09:17,363 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:09:17,365 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:09:17,366 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:09:17,372 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:09:17,373 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (null) values: [null]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:10:09,360 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Owner' on field 'entity': rejected value [null]; codes [groovyempire.Owner.entity.nullable.error.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.error.entity,groovyempire.Owner.entity.nullable.error.groovyempire.Entity,groovyempire.Owner.entity.nullable.error,owner.entity.nullable.error.groovyempire.Owner.entity,owner.entity.nullable.error.entity,owner.entity.nullable.error.groovyempire.Entity,owner.entity.nullable.error,groovyempire.Owner.entity.nullable.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.entity,groovyempire.Owner.entity.nullable.groovyempire.Entity,groovyempire.Owner.entity.nullable,owner.entity.nullable.groovyempire.Owner.entity,owner.entity.nullable.entity,owner.entity.nullable.groovyempire.Entity,owner.entity.nullable,nullable.groovyempire.Owner.entity,nullable.entity,nullable.groovyempire.Entity,nullable]; arguments [entity,class groovyempire.Owner]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrConstructorNewInstance(ReflectiveInterceptor.java:986)
	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:09,360 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Owner' on field 'entity': rejected value [null]; codes [groovyempire.Owner.entity.nullable.error.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.error.entity,groovyempire.Owner.entity.nullable.error.groovyempire.Entity,groovyempire.Owner.entity.nullable.error,owner.entity.nullable.error.groovyempire.Owner.entity,owner.entity.nullable.error.entity,owner.entity.nullable.error.groovyempire.Entity,owner.entity.nullable.error,groovyempire.Owner.entity.nullable.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.entity,groovyempire.Owner.entity.nullable.groovyempire.Entity,groovyempire.Owner.entity.nullable,owner.entity.nullable.groovyempire.Owner.entity,owner.entity.nullable.entity,owner.entity.nullable.groovyempire.Entity,owner.entity.nullable,nullable.groovyempire.Owner.entity,nullable.entity,nullable.groovyempire.Entity,nullable]; arguments [entity,class groovyempire.Owner]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:09,361 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Owner' on field 'entity': rejected value [null]; codes [groovyempire.Owner.entity.nullable.error.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.error.entity,groovyempire.Owner.entity.nullable.error.groovyempire.Entity,groovyempire.Owner.entity.nullable.error,owner.entity.nullable.error.groovyempire.Owner.entity,owner.entity.nullable.error.entity,owner.entity.nullable.error.groovyempire.Entity,owner.entity.nullable.error,groovyempire.Owner.entity.nullable.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.entity,groovyempire.Owner.entity.nullable.groovyempire.Entity,groovyempire.Owner.entity.nullable,owner.entity.nullable.groovyempire.Owner.entity,owner.entity.nullable.entity,owner.entity.nullable.groovyempire.Entity,owner.entity.nullable,nullable.groovyempire.Owner.entity,nullable.entity,nullable.groovyempire.Entity,nullable]; arguments [entity,class groovyempire.Owner]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:09,361 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Owner' on field 'entity': rejected value [null]; codes [groovyempire.Owner.entity.nullable.error.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.error.entity,groovyempire.Owner.entity.nullable.error.groovyempire.Entity,groovyempire.Owner.entity.nullable.error,owner.entity.nullable.error.groovyempire.Owner.entity,owner.entity.nullable.error.entity,owner.entity.nullable.error.groovyempire.Entity,owner.entity.nullable.error,groovyempire.Owner.entity.nullable.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.entity,groovyempire.Owner.entity.nullable.groovyempire.Entity,groovyempire.Owner.entity.nullable,owner.entity.nullable.groovyempire.Owner.entity,owner.entity.nullable.entity,owner.entity.nullable.groovyempire.Entity,owner.entity.nullable,nullable.groovyempire.Owner.entity,nullable.entity,nullable.groovyempire.Entity,nullable]; arguments [entity,class groovyempire.Owner]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:09,362 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Owner' on field 'entity': rejected value [null]; codes [groovyempire.Owner.entity.nullable.error.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.error.entity,groovyempire.Owner.entity.nullable.error.groovyempire.Entity,groovyempire.Owner.entity.nullable.error,owner.entity.nullable.error.groovyempire.Owner.entity,owner.entity.nullable.error.entity,owner.entity.nullable.error.groovyempire.Entity,owner.entity.nullable.error,groovyempire.Owner.entity.nullable.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.entity,groovyempire.Owner.entity.nullable.groovyempire.Entity,groovyempire.Owner.entity.nullable,owner.entity.nullable.groovyempire.Owner.entity,owner.entity.nullable.entity,owner.entity.nullable.groovyempire.Entity,owner.entity.nullable,nullable.groovyempire.Owner.entity,nullable.entity,nullable.groovyempire.Entity,nullable]; arguments [entity,class groovyempire.Owner]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:09,362 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Owner' on field 'entity': rejected value [null]; codes [groovyempire.Owner.entity.nullable.error.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.error.entity,groovyempire.Owner.entity.nullable.error.groovyempire.Entity,groovyempire.Owner.entity.nullable.error,owner.entity.nullable.error.groovyempire.Owner.entity,owner.entity.nullable.error.entity,owner.entity.nullable.error.groovyempire.Entity,owner.entity.nullable.error,groovyempire.Owner.entity.nullable.groovyempire.Owner.entity,groovyempire.Owner.entity.nullable.entity,groovyempire.Owner.entity.nullable.groovyempire.Entity,groovyempire.Owner.entity.nullable,owner.entity.nullable.groovyempire.Owner.entity,owner.entity.nullable.entity,owner.entity.nullable.groovyempire.Entity,owner.entity.nullable,nullable.groovyempire.Owner.entity,nullable.entity,nullable.groovyempire.Entity,nullable]; arguments [entity,class groovyempire.Owner]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:34,402 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Entity' on field 'name': rejected value [Alidad]; codes [groovyempire.Entity.name.unique.error.groovyempire.Entity.name,groovyempire.Entity.name.unique.error.name,groovyempire.Entity.name.unique.error.java.lang.String,groovyempire.Entity.name.unique.error,entity.name.unique.error.groovyempire.Entity.name,entity.name.unique.error.name,entity.name.unique.error.java.lang.String,entity.name.unique.error,groovyempire.Entity.name.unique.groovyempire.Entity.name,groovyempire.Entity.name.unique.name,groovyempire.Entity.name.unique.java.lang.String,groovyempire.Entity.name.unique,entity.name.unique.groovyempire.Entity.name,entity.name.unique.name,entity.name.unique.java.lang.String,entity.name.unique,unique.groovyempire.Entity.name,unique.name,unique.java.lang.String,unique]; arguments [name,class groovyempire.Entity,Alidad]; default message [null]

	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrConstructorNewInstance(ReflectiveInterceptor.java:986)
	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Entity$$EOOpzFJY.establish(Entity.groovy:17)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:34,402 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Entity' on field 'name': rejected value [Alidad]; codes [groovyempire.Entity.name.unique.error.groovyempire.Entity.name,groovyempire.Entity.name.unique.error.name,groovyempire.Entity.name.unique.error.java.lang.String,groovyempire.Entity.name.unique.error,entity.name.unique.error.groovyempire.Entity.name,entity.name.unique.error.name,entity.name.unique.error.java.lang.String,entity.name.unique.error,groovyempire.Entity.name.unique.groovyempire.Entity.name,groovyempire.Entity.name.unique.name,groovyempire.Entity.name.unique.java.lang.String,groovyempire.Entity.name.unique,entity.name.unique.groovyempire.Entity.name,entity.name.unique.name,entity.name.unique.java.lang.String,entity.name.unique,unique.groovyempire.Entity.name,unique.name,unique.java.lang.String,unique]; arguments [name,class groovyempire.Entity,Alidad]; default message [null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Entity$$EOOpzFJY.establish(Entity.groovy:17)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:34,403 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Entity' on field 'name': rejected value [Alidad]; codes [groovyempire.Entity.name.unique.error.groovyempire.Entity.name,groovyempire.Entity.name.unique.error.name,groovyempire.Entity.name.unique.error.java.lang.String,groovyempire.Entity.name.unique.error,entity.name.unique.error.groovyempire.Entity.name,entity.name.unique.error.name,entity.name.unique.error.java.lang.String,entity.name.unique.error,groovyempire.Entity.name.unique.groovyempire.Entity.name,groovyempire.Entity.name.unique.name,groovyempire.Entity.name.unique.java.lang.String,groovyempire.Entity.name.unique,entity.name.unique.groovyempire.Entity.name,entity.name.unique.name,entity.name.unique.java.lang.String,entity.name.unique,unique.groovyempire.Entity.name,unique.name,unique.java.lang.String,unique]; arguments [name,class groovyempire.Entity,Alidad]; default message [null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Entity$$EOOpzFJY.establish(Entity.groovy:17)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:34,403 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Entity' on field 'name': rejected value [Alidad]; codes [groovyempire.Entity.name.unique.error.groovyempire.Entity.name,groovyempire.Entity.name.unique.error.name,groovyempire.Entity.name.unique.error.java.lang.String,groovyempire.Entity.name.unique.error,entity.name.unique.error.groovyempire.Entity.name,entity.name.unique.error.name,entity.name.unique.error.java.lang.String,entity.name.unique.error,groovyempire.Entity.name.unique.groovyempire.Entity.name,groovyempire.Entity.name.unique.name,groovyempire.Entity.name.unique.java.lang.String,groovyempire.Entity.name.unique,entity.name.unique.groovyempire.Entity.name,entity.name.unique.name,entity.name.unique.java.lang.String,entity.name.unique,unique.groovyempire.Entity.name,unique.name,unique.java.lang.String,unique]; arguments [name,class groovyempire.Entity,Alidad]; default message [null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Entity$$EOOpzFJY.establish(Entity.groovy:17)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:34,403 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Entity' on field 'name': rejected value [Alidad]; codes [groovyempire.Entity.name.unique.error.groovyempire.Entity.name,groovyempire.Entity.name.unique.error.name,groovyempire.Entity.name.unique.error.java.lang.String,groovyempire.Entity.name.unique.error,entity.name.unique.error.groovyempire.Entity.name,entity.name.unique.error.name,entity.name.unique.error.java.lang.String,entity.name.unique.error,groovyempire.Entity.name.unique.groovyempire.Entity.name,groovyempire.Entity.name.unique.name,groovyempire.Entity.name.unique.java.lang.String,groovyempire.Entity.name.unique,entity.name.unique.groovyempire.Entity.name,entity.name.unique.name,entity.name.unique.java.lang.String,entity.name.unique,unique.groovyempire.Entity.name,unique.name,unique.java.lang.String,unique]; arguments [name,class groovyempire.Entity,Alidad]; default message [null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Entity$$EOOpzFJY.establish(Entity.groovy:17)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:10:34,404 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.Entity' on field 'name': rejected value [Alidad]; codes [groovyempire.Entity.name.unique.error.groovyempire.Entity.name,groovyempire.Entity.name.unique.error.name,groovyempire.Entity.name.unique.error.java.lang.String,groovyempire.Entity.name.unique.error,entity.name.unique.error.groovyempire.Entity.name,entity.name.unique.error.name,entity.name.unique.error.java.lang.String,entity.name.unique.error,groovyempire.Entity.name.unique.groovyempire.Entity.name,groovyempire.Entity.name.unique.name,groovyempire.Entity.name.unique.java.lang.String,groovyempire.Entity.name.unique,entity.name.unique.groovyempire.Entity.name,entity.name.unique.name,entity.name.unique.java.lang.String,entity.name.unique,unique.groovyempire.Entity.name,unique.name,unique.java.lang.String,unique]; arguments [name,class groovyempire.Entity,Alidad]; default message [null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.Entity$$EOOpzFJY.establish(Entity.groovy:17)
	at groovyempire.Owner$$EOOpz8YC.establish(Owner.groovy:7)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:28)
2013-12-02 00:11:23,110 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:11:23,111 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:11:23,114 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:11:23,115 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:11:23,121 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:11:23,123 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOpxxBo.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:35,184 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:35,186 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:35,189 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:35,190 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:35,196 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:35,198 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:43,272 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:43,273 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:43,275 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:43,277 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:43,280 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:13:43,281 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOpyvHc.establish(Account.groovy:24)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:15:03,349 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOq0NIG.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:15:03,350 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOq0NIG.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:15:03,353 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOq0NIG.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:15:03,355 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOq0NIG.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:15:03,360 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOq0NIG.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:15:03,362 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: (groovyempire.Owner) values: [groovyempire.Owner : 2]
Possible solutions: getOwners()
	at groovyempire.banking.Account$$EOOq0NIG.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:12,196 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: () values: []
Possible solutions: getOwners(), setOwners(java.util.Set)
	at groovyempire.banking.Account$$EOOq0eiu.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:12,198 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: () values: []
Possible solutions: getOwners(), setOwners(java.util.Set)
	at groovyempire.banking.Account$$EOOq0eiu.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:12,201 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: () values: []
Possible solutions: getOwners(), setOwners(java.util.Set)
	at groovyempire.banking.Account$$EOOq0eiu.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:12,202 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: () values: []
Possible solutions: getOwners(), setOwners(java.util.Set)
	at groovyempire.banking.Account$$EOOq0eiu.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:12,208 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: () values: []
Possible solutions: getOwners(), setOwners(java.util.Set)
	at groovyempire.banking.Account$$EOOq0eiu.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:12,209 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingMethodException: No signature of method: groovyempire.banking.Account.addToOwners() is applicable for argument types: () values: []
Possible solutions: getOwners(), setOwners(java.util.Set)
	at groovyempire.banking.Account$$EOOq0eiu.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:17,375 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrConstructorNewInstance(ReflectiveInterceptor.java:986)
	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq0gXo.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:17,376 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq0gXo.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:17,376 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq0gXo.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:17,376 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq0gXo.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:17,377 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq0gXo.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:16:17,377 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq0gXo.establish(Account.groovy:21)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:17:35,428 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrConstructorNewInstance(ReflectiveInterceptor.java:986)
	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.springsource.loaded.ri.ReflectiveInterceptor.jlrMethodInvoke(ReflectiveInterceptor.java:1260)
	at groovy.lang.Closure.call(Closure.java:411)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq10Zk.establish(Account.groovy:26)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:17:35,428 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq10Zk.establish(Account.groovy:26)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:17:35,428 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq10Zk.establish(Account.groovy:26)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:17:35,428 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq10Zk.establish(Account.groovy:26)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:17:35,429 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq10Zk.establish(Account.groovy:26)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:17:35,429 [main] ERROR StackTrace  - Full Stack Trace:
grails.validation.ValidationException: Validation error occurred during call to save():

- Field error in object 'groovyempire.banking.Account' on field 'bank': rejected value [null]; codes [groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.error.bank,groovyempire.banking.Account.bank.nullable.error.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable.error,account.bank.nullable.error.groovyempire.banking.Account.bank,account.bank.nullable.error.bank,account.bank.nullable.error.groovyempire.banking.Bank,account.bank.nullable.error,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Account.bank,groovyempire.banking.Account.bank.nullable.bank,groovyempire.banking.Account.bank.nullable.groovyempire.banking.Bank,groovyempire.banking.Account.bank.nullable,account.bank.nullable.groovyempire.banking.Account.bank,account.bank.nullable.bank,account.bank.nullable.groovyempire.banking.Bank,account.bank.nullable,nullable.groovyempire.banking.Account.bank,nullable.bank,nullable.groovyempire.banking.Bank,nullable]; arguments [bank,class groovyempire.banking.Account]; default message [Property [{0}] of class [{1}] cannot be null]

	at org.grails.datastore.gorm.GormInstanceApi.doSave(GormInstanceApi.groovy:182)
	at org.grails.datastore.gorm.GormInstanceApi.save_closure5(GormInstanceApi.groovy:162)
	at org.grails.datastore.mapping.core.DatastoreUtils.execute(DatastoreUtils.java:302)
	at org.grails.datastore.gorm.AbstractDatastoreApi.execute(AbstractDatastoreApi.groovy:37)
	at org.grails.datastore.gorm.GormInstanceApi.save(GormInstanceApi.groovy:161)
	at groovyempire.banking.Account$$EOOq10Zk.establish(Account.groovy:26)
	at groovyempire.banking.Bank$$EOOq008m.establishAccount(Bank.groovy:28)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:23:55,124 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:23:55,128 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:23:55,129 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:23:55,129 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:23:55,186 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:23:55,186 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank.establishAccount(Bank.groovy:29)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:19,582 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOq2x1U.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:19,583 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOq2x1U.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:19,583 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOq2x1U.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:19,583 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOq2x1U.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:19,584 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOq2x1U.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:19,584 [main] ERROR StackTrace  - Full Stack Trace:
java.lang.NullPointerException: Cannot invoke method leftShift() on null object
	at groovyempire.banking.Bank$$EOOq2x1U.establishAccount(Bank.groovy:32)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:48,085 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'groovyempire.banking.Bank : 1' with class 'groovyempire.banking.Bank' to class 'groovyempire.banking.Account'
	at groovyempire.banking.Bank$$EOOq34ZE.establishAccount(Bank.groovy:33)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:48,085 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'groovyempire.banking.Bank : 1' with class 'groovyempire.banking.Bank' to class 'groovyempire.banking.Account'
	at groovyempire.banking.Bank$$EOOq34ZE.establishAccount(Bank.groovy:33)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:48,085 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'groovyempire.banking.Bank : 1' with class 'groovyempire.banking.Bank' to class 'groovyempire.banking.Account'
	at groovyempire.banking.Bank$$EOOq34ZE.establishAccount(Bank.groovy:33)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:48,085 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'groovyempire.banking.Bank : 1' with class 'groovyempire.banking.Bank' to class 'groovyempire.banking.Account'
	at groovyempire.banking.Bank$$EOOq34ZE.establishAccount(Bank.groovy:33)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:48,086 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'groovyempire.banking.Bank : 1' with class 'groovyempire.banking.Bank' to class 'groovyempire.banking.Account'
	at groovyempire.banking.Bank$$EOOq34ZE.establishAccount(Bank.groovy:33)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:25:48,086 [main] ERROR StackTrace  - Full Stack Trace:
org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'groovyempire.banking.Bank : 1' with class 'groovyempire.banking.Bank' to class 'groovyempire.banking.Account'
	at groovyempire.banking.Bank$$EOOq34ZE.establishAccount(Bank.groovy:33)
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:30)
2013-12-02 00:27:21,320 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: amount for class: groovyempire.banking.Account
Possible solutions: count
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:33)
2013-12-02 00:27:21,377 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: amount for class: groovyempire.banking.Account
Possible solutions: count
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:33)
2013-12-02 00:27:21,388 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: amount for class: groovyempire.banking.Account
Possible solutions: count
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:33)
2013-12-02 00:27:21,390 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: amount for class: groovyempire.banking.Account
Possible solutions: count
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:33)
2013-12-02 00:27:21,410 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: amount for class: groovyempire.banking.Account
Possible solutions: count
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:33)
2013-12-02 00:27:21,413 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: amount for class: groovyempire.banking.Account
Possible solutions: count
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:33)
2013-12-02 00:28:13,528 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: accountOnwer for class: groovyempire.AccountSpec
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:13,529 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: accountOnwer for class: groovyempire.AccountSpec
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:13,529 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: accountOnwer for class: groovyempire.AccountSpec
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:13,529 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: accountOnwer for class: groovyempire.AccountSpec
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:13,530 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: accountOnwer for class: groovyempire.AccountSpec
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:13,530 [main] ERROR StackTrace  - Full Stack Trace:
groovy.lang.MissingPropertyException: No such property: accountOnwer for class: groovyempire.AccountSpec
	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:32,028 [main] ERROR StackTrace  - Full Stack Trace:
Condition not satisfied:

account.owners  == [accountOwner]
|       |       |   |
|       |       |   groovyempire.Owner : 2
|       |       false
|       [groovyempire.Owner : 2]
groovyempire.banking.Account : 1

	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:32,032 [main] ERROR StackTrace  - Full Stack Trace:
Condition not satisfied:

account.owners  == [accountOwner]
|       |       |   |
|       |       |   groovyempire.Owner : 2
|       |       false
|       [groovyempire.Owner : 2]
groovyempire.banking.Account : 1

	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:32,032 [main] ERROR StackTrace  - Full Stack Trace:
junit.framework.AssertionFailedError: Condition not satisfied:

account.owners  == [accountOwner]
|       |       |   |
|       |       |   groovyempire.Owner : 2
|       |       false
|       [groovyempire.Owner : 2]
groovyempire.banking.Account : 1

	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:32,032 [main] ERROR StackTrace  - Full Stack Trace:
junit.framework.AssertionFailedError: Condition not satisfied:

account.owners  == [accountOwner]
|       |       |   |
|       |       |   groovyempire.Owner : 2
|       |       false
|       [groovyempire.Owner : 2]
groovyempire.banking.Account : 1

	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:32,033 [main] ERROR StackTrace  - Full Stack Trace:
junit.framework.AssertionFailedError: Condition not satisfied:

account.owners  == [accountOwner]
|       |       |   |
|       |       |   groovyempire.Owner : 2
|       |       false
|       [groovyempire.Owner : 2]
groovyempire.banking.Account : 1

	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
2013-12-02 00:28:32,033 [main] ERROR StackTrace  - Full Stack Trace:
junit.framework.AssertionFailedError: Condition not satisfied:

account.owners  == [accountOwner]
|       |       |   |
|       |       |   groovyempire.Owner : 2
|       |       false
|       [groovyempire.Owner : 2]
groovyempire.banking.Account : 1

	at groovyempire.AccountSpec.Create account for a bank(AccountSpec.groovy:34)
